src/Action/Server.hs

```diff
@@ -1,7 +1,10 @@
 {-# LANGUAGE ViewPatterns, TupleSections, RecordWildCards, ScopedTypeVariables, PatternGuards #-}
 {-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE MultiWayIf #-}
+{-# OPTIONS_GHC -Wall #-}
+{-# LANGUAGE NamedFieldPuns #-}
 
-module Action.Server(actionServer, actionReplay, action_server_test_, action_server_test) where
+module Action.Server(actionServer, actionReplay, action_server_test_, action_server_test, takeAndGroup) where
```

Comment:
Why export takeAndGroup? It doesn't look like it's used?

Reply:
Ah, yeah, that was just to try it out in the repl